src/test/rgw/test_rgw_amqp_1.cc

```diff
@@ -0,0 +1,102 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+
+#include "rgw/rgw_amqp_1.h"
+#include "common/ceph_context.h"
+#include <chrono>
+#include <gtest/gtest.h>
+
+#include <iostream>
+
+/*
+
+1. first build the amqp 1.0 broker, then run locally specifying a [address:port/topic]
+  e.g.  # ${BROKER_BINARY} -a localhost:5672/amqp1_0
```

Comment:
please specify the binary name - since we are building it, we should have the name